### PR TITLE
Update MBS feature page

### PR DESCRIPTION
**Pull request by rollandf**
Changes proposed in this pull request:

- Update VDSM APIs and update the page.

I confirm that this pull request was submitted according to the [contribution guidelines](https://github.com/oVirt/ovirt-site/blob/master/CONTRIBUTING.md): @rollandf 

This pull request needs review by: @shenitzky @bennyz

### CODE REVIEW

File: source/develop/release-management/features/storage/cinderlib-integration.html.md [hunk 1]
```diff
@@ -24,13 +24,13 @@ This feature enables the user to be able to consume any storage backend supporte
...
-Cinder Library [1] provides a Python library that give the possibility to use the Cinder drivers directly without the need of a full Openstack/Cinder deployment supporting over 80 storage drivers.
+Cinder Library [1] provides a Python library that give the possibility to use the Cinder drivers directly without the need of a full OpenStack/Cinder deployment supporting over 80 storage drivers.
 
 - The user is able to configure a Storage Domain with the name of the Cinder driver and a set of parameters needed by the driver implementation.
-- The user is able to create/delete disks on that Storage Domain.
+- The user is able to create/delete/clone disks on that Storage Domain.
 - The user is able to create/delete snapshots of the disks created on that Storage Domain.
 - The user is able to run VMs with disks created on that Storage Domain.
-
+- The user is able to create templates from VMs with disks created on that Storage Domain.
```
Comment (shenitzky) [inline source/develop/release-management/features/storage/cinderlib-integration.html.md]: The user is able to create a VM from a template with disks on that storage domain

File: source/develop/release-management/features/storage/cinderlib-integration.html.md [hunk 2]
```diff
@@ -83,45 +124,142 @@ The user delete a disk on the storage domain:
...
 
-The Vdsm uses the Cinder's 'os_brick' library to collect the data.
-
-
 #### Run VM
 
 The user runs a VM with a Cinderlib disk :
-  - The Engine calls a Cinderlib API to connect the volume using the connection information for the specific host and return the needed data for the attachment.
+  - The Engine calls a Cinderlib API to connect the volume using the connector information for the specific host and return the needed data for the attachment.
   - The Engine calls a Vdsm verb to attach the volume with the data needed. The Vdsm stores locally the attachment details that will be needed for detach, and then return the path.
```
Comment (shenitzky) [inline source/develop/release-management/features/storage/cinderlib-integration.html.md]: needed data

Comment (shenitzky) [inline source/develop/release-management/features/storage/cinderlib-integration.html.md]: s/Vdsm/host

File: source/develop/release-management/features/storage/cinderlib-integration.html.md [hunk 3]
```diff
@@ -83,45 +124,142 @@ The user delete a disk on the storage domain:
...
-  - The Engine uses the returned path for the volume in the VM XML.
+  - The Engine stores the result in the DB and uses the returned path for the volume in the VM XML.
 
 
 #### Stop VM
 
 The user stops a VM with a Cinderlib disk :
-  - The Engine calls a Vdsm verb to detach the volume with the data needed. The Vdsm use the attachment details provided by the Engine and disconnect the volume.
+  - The Engine calls a Vdsm verb to detach the volume with the volume ID. The Vdsm use the ID provided by the Engine, fetches the needed data from its local DB and disconnect the volume.
```
Comment (shenitzky) [inline source/develop/release-management/features/storage/cinderlib-integration.html.md]: s/Vdsm/host

File: source/develop/release-management/features/storage/cinderlib-integration.html.md [hunk 4]
```diff
@@ -83,45 +124,142 @@ The user delete a disk on the storage domain:
...
+        "target_lun": 26,
+        "target_iqn": "iqn.2009-01.com.kaminario:storage.k2.22612",
+        "target_portal": "3.2.1.1:3260",
+        "target_discovered": "True"
+    }
+}
+```
+
+Vdsm will call an os_brick API to attach the volume and will return the volume information data to Engine that will persist it in the DB.
+The volume information consists of the volume path that Engine will use in the VM XML and of the volume attachment as returned from os_brick. The structure of the volume attachment data is according to os_brick/Cinder
```
Comment (shenitzky) [inline source/develop/release-management/features/storage/cinderlib-integration.html.md]: already written in line 171

File: source/develop/release-management/features/storage/cinderlib-integration.html.md (hunk 1 above)
Comment (rollandf) [reply source/develop/release-management/features/storage/cinderlib-integration.html.md]: Done

File: source/develop/release-management/features/storage/cinderlib-integration.html.md (hunk 2 above)
Comment (rollandf) [reply source/develop/release-management/features/storage/cinderlib-integration.html.md]: Done

Comment (rollandf) [reply source/develop/release-management/features/storage/cinderlib-integration.html.md]: I think that 'Vdsm' is better

File: source/develop/release-management/features/storage/cinderlib-integration.html.md (hunk 4 above)
Comment (rollandf) [reply source/develop/release-management/features/storage/cinderlib-integration.html.md]: This is referring to 2 differents data object. It is important to mention it for both

**Pull request merged**